vendor/src/github.com/docker/libnetwork/portmapper/mapper.go

```diff
@@ -128,8 +128,10 @@ func (pm *PortMapper) MapRange(container net.Addr, hostIP net.IP, hostPortStart,
 	}
 
 	containerIP, containerPort := getIPAndPort(m.container)
-	if err := pm.forward(iptables.Append, m.proto, hostIP, allocatedHostPort, containerIP.String(), containerPort); err != nil {
-		return nil, err
+	if hostIP.To4() != nil {
+		if err := pm.forward(iptables.Append, m.proto, hostIP, allocatedHostPort, containerIP.String(), containerPort); err != nil {
+			return nil, err
+		}
```

Comment:
When you submit to libnetwork, remember to also add the `.To4()` check around the `pm.forward()` in the following defer statement at line 138.